packages/paste-design-tokens/tokens/aliases/font-family.yml

```diff
@@ -1,3 +1,7 @@
 aliases:
   font-family-10: "'Inter var experimental', 'Inter var', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif"
   font-family-20: "'Fira Mono', 'Courier New', Courier, monospace"
+  font-family-ja: "'Inter var experimental', 'Inter var', Hiragino Sans, 'ヒラギノ角ゴ ProN W3', 'Hiragino Kaku Gothic ProN', 'メイリオ', Meiryo, Osaka, 'MS PGothic', sans-serif"
+  font-family-ko: "'Inter var experimental', 'Inter var', 'Microsoft JhengHei', 微軟正黑體, 'Microsoft JhengHei UI', 'Microsoft YaHei', 微軟雅黑, 宋体, SimSun, sans-serif"
+  font-family-zh-traditional: "'Inter var experimental', 'Inter var', 'Microsoft JhengHei', 微軟正黑體, 'Microsoft JhengHei UI', 'Microsoft YaHei', 微軟雅黑, 宋体, SimSun, sans-serif"
+  font-family-zh-simplified: "'Inter var experimental', 'Inter var', 'Microsoft YaHei New', 微软雅黑, 'Microsoft Yahei', 'Microsoft JhengHei', 宋体, SimSun, sans-serif"
```

Comment:
Question: The selector for this one is `html:lang(zh-CN)`. Should it be `font-family-zh-cn`? This also depends on the answer to the 2 above questions.

Reply:
see above ^